pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/design/xml/LawOfDemeter.xml

```diff
@@ -269,6 +269,20 @@ public class Test {
         final FooBuilder fooBuilder = FooBuilder.newBuilder();
         fooBuilder.withBar();
     }
+}
+        ]]></code>
+    </test-code>
+    <test-code>
+        <description>#1429 False Positive for Law of Demeter</description>
```

Comment:
To which issue report are you refering? #1429 seems to be unrelated to LawOfDemeterRule...

I'd put a reference to this pull request instead...